lib/auth/auth_test.go

```diff
@@ -725,19 +745,22 @@ func (s *AuthSuite) TestGenerateTokenEventsEmitted(c *C) {
 		types.RoleTrustedCluster,
 		types.RoleAuth,
 	}})
-	c.Assert(err, IsNil)
-	c.Assert(s.mockEmitter.LastEvent().GetType(), DeepEquals, events.TrustedClusterTokenCreateEvent)
+	require.NoError(t, err)
+	require.Equal(t, s.mockEmitter.LastEvent().GetType(), events.TrustedClusterTokenCreateEvent)
 }
 
-func (s *AuthSuite) TestValidateACRValues(c *C) {
+func TestValidateACRValues(t *testing.T) {
+	s := newAuthSuite(t)
+
 	tests := []struct {
+		comment       string
```

Comment:
Suggestion: You could pass in `require.Error` or `require.NoError` and remove the if from the test below
```suggestion
		outIsValid    require.ErrorAssertionFunc
```